ExampleMod/Content/Projectiles/ExampleAdvancedFlailProjectile.cs

```diff
@@ -39,6 +42,16 @@ private enum AIState
 		public ref float CollisionCounter => ref Projectile.localAI[0];
 		public ref float SpinningStateTimer => ref Projectile.localAI[1];
 
+		public override void Load() {
+			chainTexture = ModContent.Request<Texture2D>(ChainTexturePath);
+			chainTextureExtra = ModContent.Request<Texture2D>(ChainTextureExtraPath);
+		}
```

Comment:
Sometimes we `Request` on `Load`, sometimes on demand. We should give some advice, or pick one and stick to it

Reply:
Most are on load now, I think that is best.